### ISSUE TITLE
"Bad Luck Protection" for Artifacts

### ISSUE THREAD

**Issue by Fynzou** [StardewModders/mod-ideas#84]
Completing the Museum is entirely luck based at the moment. I'd love to have a mod that took the luck out of the equation, at least to some extent. 

For instance: Every artifact you dig up you've already found increases the chance of finding an artifact you've never found until you're guaranteed to get one you haven't found after an indeterminate amount.

**Comment by Entoarox** [comment]
Due to how artifact digging is handled in the code, this while possible would require Harmony - and thus is automatically a difficult enough task to be given the `hard` label.

**Comment by Pathoschild** [comment]
Not necessarily. Worst case, the mod can detect which artifact was just dug up and swap it.

**Comment by tiakall** [comment]
Vanilla Plus Professions has a talent that does this! https://www.nexusmods.com/stardewvalley/mods/20054
